linkchecker

```diff
@@ -42,13 +43,13 @@ from linkcheck.cmdline import (
     LCArgumentParser,
     print_plugins,
 )
-from linkcheck import log, i18n, strformat
```

Comment:
This is slightly iffy: if `linkcheck.log` is a module, then `import linkcheck` doesn't guarantee that `linkcheck.log` will be imported and available, unless `linkcheck/__init__.py` contains an `import .log` somewhere.  (Which it does, but will that always be the case?  Probably.)

As usual I'm splitting hairs for no good reason.  It's fine.

Reply:
Keep splitting! Rewritten to use `from linkcheck import log, logconf` and `from linkcheck.i18n import default_encoding`

Reply:
Don't think `from linkcheck.i18n import default_encoding` is correct. default_encoding is set as a global in i18n.init() which is called from linkcheck/__init__.py...